src/org/omegat/Bundle.properties

```
@@ -504,8 +504,10 @@ Reload the project now?
 
 TF_SOURCE_LOAD_ERROR=Skipped source file {0} because it could not be loaded.
 
-TF_TM_LOAD_ERROR=Failed to load translation memory {0} for project!
-TF_LOAD_ERROR=Failed to load specified project!
+TF_TM_LOAD_ERROR=Failed to load translation memory {0} for the project!
+TF_LOAD_ERROR=Failed to load the specified project!
+TF_LOAD_ERROR_PARSER_EXCEPTION=Failed to load the project by corruption of files. Please check omegat directory of \
+  project, there are healthy filters.xml and segmentation.srx exits.
```

filter loader and segment loader does not raise XML parser error. so we should change message.
```suggestion
TF_LOAD_ERROR_PARSER_EXCEPTION=Failed to load the project by corruption of files. Please check omegat directory of  the project.
```

---

Thank you.